testing_and_setup/compass/ocean/overflow/1km/rpe_test/config_rpe_test_6_nu_1000.xml

```diff
@@ -9,7 +9,8 @@
 		<option name="config_dt">'0000_00:03:00'</option>
 		<option name="config_btr_dt">'0000_00:00:09'</option>
 		<option name="config_run_duration">'0000_40:00:00'</option>
-		<option name="config_use_const_visc">.true.</option>
```

Comment:
Too bad this all can't be lumped together in a template file-- don't think it is worth effort here for now.